docker-entrypoint.sh

```diff
@@ -126,19 +126,22 @@ if [ "$1" = 'rabbitmq-server' ]; then
 			].
 		EOF
 	fi
+fi
 
-	if [ "$ssl" ]; then
-		# Create combined cert
-		cat "$RABBITMQ_SSL_CERT_FILE" "$RABBITMQ_SSL_KEY_FILE" > /tmp/combined.pem
-		chmod 0400 /tmp/combined.pem
+if [ "$ssl" ]; then
+	# Create combined cert
+	# If calling this script with an exec this is problematic, as it needs to be owned 400 for perms but this code can run as root OR rabbitmq.
+	# As a workaround prefix the combined cert with the username so each user only operates on the file they control.
+	combined_file="/tmp/$(id -un)_combined.pem"
```

Comment:
If we just adjust [line 5](https://github.com/docker-library/rabbitmq/blob/e9e416f99643363b9a9195929568909eadf8ce40/docker-entrypoint.sh#L5) to also switch users from root when running `rabbitmqctl` then the tmp file with identical contents would not have to made for each user.  This wouldn't help them when running bash, but that feels more like a documentation problem to get them to run `rabbitmqctl` as the same user that the server is running as.

As is, this current line would fail if a user is running rabbit as a user id that does not exist in the container (which is valid).
```console
$ docker run -d --user 1000:50 -v /Users/rabbit/:/var/lib/rabbitmq --name rabbit --hostname my-rabbit rabbitmq
# of course adding in the ssl stuff too
$ docker exec -it rabbit rabbitmqctl status
```